### PR TITLE
Add Github profile link to username

### PR DESCRIPTION
**Pull request by abhisheksatre**
Github profile link of the searched user is added so the user can see his/her projects.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -46,6 +46,10 @@ const App = () => {
     }
   }
 
+  const getProfileLink = ()=>{
+      return <a href={`https://github.com/${login}`} rel="noopener noreferrer" target="_blank" >{name || login}</a>;
+  }
+
```
Comment (NomanGul) [inline src/App.js:54]: good addition though... Could you please make a link color dull? `cause right now, It's not matching to our theme

Comment (NomanGul) [reply src/App.js:54]: @abhisheksatre

Comment (abhisheksatre) [reply src/App.js:54]: @NomanGul light black color added to match with the theme

**Comment by NomanGul** [comment]
@abhisheksatre also can you please change the base branch from master to development?

**Comment by NomanGul** [comment]
@abhisheksatre Thanks for this cool contribution and sorry for wasting your time :rocket:

**Comment by abhisheksatre** [comment]
@NomanGul Thank you. no issues 👍

**Pull request merged**